tests/foreman/cli/test_repository.py

```diff
@@ -1659,7 +1659,8 @@ def test_negative_restricted_user_cv_add_repository(self):
         with self.assertRaises(CLIReturnCodeError) as context:
             Role.with_user(user_name, user_password).info({'id': role['id']})
         self.assertIn(
-            'Forbidden - server refused to process the request', context.exception.stderr
+            'Access denied\n' 'Missing one of the required permissions: view_roles\n',
```

Comment:
The way it's written is little weird. Couldn't that be a single string?
Also, this last new line character - is it required?

Reply:
Its was black that reformatted that. OK, fixed, 

Thank you